eth/main.cpp

```diff
@@ -349,7 +349,7 @@ int main(int argc, char** argv)
 	generalOptions.add_options()
 		("db-path,d", po::value<string>()->value_name("<path>"), ("Load database from path\n(default: " + getDataDir().string() + ").\n").c_str())
 #if ETH_EVMJIT
-		("vm", "<vm-kind>  Select VM; options are: interpreter, jit or smart (default: interpreter)")
+		("vm", po::value<string>()->value_name("<n>"), "<vm-kind>  Select VM; options are: interpreter, jit or smart (default: interpreter)")
```

Comment:
I think it should be `value_name("<vm-kind>")` and the `<vm-kind>` text be removed from the help text.